### PR TITLE
Add icon for ul, ol and code block

### PR DESCRIPTION
**Pull request by pronayguha13**
Added icon in place of Text in the Editor.

### CODE REVIEW

File: src/pages/Home/components/Editor/index.js [hunk 1]
```diff
@@ -2,6 +2,9 @@ import React, { useState, useEffect } from "react";
 
 import "./style/index.css";
 
+import ul from "../../../../assets/icons/unordered-list.png";
+import ol from "../../../../assets/icons/numbered-list.png";
+import code from "../../../../assets/icons/code-block.png";
```
Comment (Jaagrav) [inline src/pages/Home/components/Editor/index.js:7]: So I am using Material UI for this project, if you can please import a suitable icon from @material-ui/icons instead.

File: src/pages/Home/components/Editor/index.js [hunk 2]
```diff
@@ -214,9 +217,15 @@ function Editor({ match }) {
...
-        <Button onMouseDown={onCodeBlockClick}>Code Block</Button>
+        <Button onMouseDown={onULClick}>
+          <img src={ul} alt="UL" />
+        </Button>
+        <Button onMouseDown={onOLClick}>
+          <img src={ol} alt="OL" />
+        </Button>
+        <Button onMouseDown={onCodeBlockClick}>
+          <img src={code} alt="Code Block" />
+        </Button>
```
Comment (Jaagrav) [inline src/pages/Home/components/Editor/index.js:228]: You can look for suitable icons [here](https://material-ui.com/components/material-icons/)

**Review by Jaagrav — CHANGES_REQUESTED** [review]
Use material icons instead, check diff comments.

File: src/pages/Home/components/Editor/index.js (hunk 1 above)
Comment (pronayguha13) [reply src/pages/Home/components/Editor/index.js:7]: Okay sure I'll definitely do the needful

**Comment by pronayguha13** [comment]
Added icons from material-ui but there may be some issue related to git.
Please check them first.

**Pull request closed without merging**